astropy/units/tests/test_quantity_annotations.py

```diff
@@ -1,9 +1,112 @@
 # -*- coding: utf-8 -*-
 # Licensed under a 3-clause BSD style license - see LICENSE.rst
 
+# STDLIB
+import typing as T
+
+# THIRD PARTY
 import pytest
 
+# LOCAL
 from astropy import units as u  # pylint: disable=W0611
```

Comment:
Nitpick, but why the pylint here? If `u` is not used, just remove the import?

Reply:
I can remove this. This must be old. I didn't put this here.

Reply:
Done!